docs/supermarket-release-checklist.md

```diff
@@ -0,0 +1,100 @@
+### Testing on ubuntu 20.04.2 LTS
+
+- Build supermarket artifact with omnibus
+  - Go to the supermarket repo. Inside repo go to `omnibus` directory
+  - run `bundle install --binstubs`
+
+- Install supermarket artifact
+  - Once build is successful it will create a supermarket installation package in the directory `/var/cache/omnibus/pkg`.
+    U can install the package using respective package installer for the selected OS.
+

```

Comment:
Could be better if we provide command based instruction here e.g. `dpkg -I /var/cache/omnibus/pkg/<pkg>` as we are providing instruction for Ubuntu

Reply:
> Could be better if we provide command based instruction here e.g. `dpkg -I /var/cache/omnibus/pkg/<pkg>` as we are providing instruction for Ubuntu

Added the command.